.changeset/slimy-items-happen.md

```diff
@@ -0,0 +1,14 @@
+---
+'@swisspost/design-system-styles': major
+---
+
+Updated CSS and SCSS entry file names to better represent the styles contained. There is now the choice between compact and default appearance instead of internal and external styles. This change better represents the resulting look of the components.
+
+- post-external.[s]css -> post-default.[s]css
```

Comment:
Didn't we define that we would simply use `post.[s]css` for the entry file?

Reply:
I tried to apply this naming pattern, but I found it leads to more confusion than it saves on typing or cleanlyness. For example in the imports:

```scss
@layer design-system {
  @include meta.load-css('./tokens/schemes');
  @include meta.load-css('./tokens/device-default');
  @include meta.load-css('./tokens/appearance-default');
  @include meta.load-css('./tokens/post-theme');
}
```

If the device and appearance imports are not named "default", it looks like we're importing all device and appearance tokens and not only the default set. As a consequence of this, also the main import files have the default postfix. What do you think?

Reply:
That make sense